src/awsexplorer/regionNode.ts

```diff
@@ -25,6 +25,7 @@ import { DefaultEcrClient } from '../shared/clients/ecrClient'
 import { DefaultIotClient } from '../shared/clients/iotClient'
 import { DefaultS3Client } from '../shared/clients/s3Client'
 import { DefaultSchemaClient } from '../shared/clients/schemaClient'
+import { getCodeArtifactRootNode } from '../codeArtifact/model'
```

Comment:
```suggestion
import { getCodeArtifactRootNode } from '../codeartifact/model'
```

Reply:
There's a TS config setting to check for these kinds of issues but it seems to slow down type checking significantly :/